docs/source/optimizations/ibm/ibmq_compile_qss.ipynb

```diff
@@ -21,9 +21,37 @@
     "</a>"
```

Comment:
<p>Should we just use qss.SuperstaQProvider() to stay close to the css.Service() in the css notebook?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/539/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Good catch -- done.